### PR TITLE
fix: add jakartaee-web-api test dependency

### PR DESCRIPTION
**Pull request by tomivirkki**
Fix the `ClassesSerializableTest` test failures

### CODE REVIEW

File: vaadin-accordion-flow-parent/vaadin-accordion-flow/pom.xml [hunk 1]
```diff
@@ -44,6 +44,12 @@
             <artifactId>slf4j-simple</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.platform</groupId>
+            <artifactId>jakarta.jakartaee-web-api</artifactId>
+            <version>10.0.0</version>
```
Comment (knoobie) [inline vaadin-accordion-flow-parent/vaadin-accordion-flow/pom.xml:50]: Wanna add this versioning (dependencyManagement) to the root pom like done in flow to ease migrating later to 11?

Comment (tomivirkki) [reply vaadin-accordion-flow-parent/vaadin-accordion-flow/pom.xml]: Done

**Review by vursen — APPROVED** [review]

**Comment by knoobie** [comment]
The same is needed for v24 as well or?

**Comment by vursen** [comment]
> The same is needed for v24 as well or?

The [related Flow fix](https://github.com/vaadin/flow/pull/16223) was back-ported to 24.0, so good catch. Let me add a 24.0 label.

**Pull request merged**